examples/red_teaming/redteaming_simple_calculator/README.md

```
@@ -0,0 +1,312 @@
+
+## Architecture
+
+The red teaming evaluation system consists of:
+
+1. **Base Workflow Config**: A standard NeMo Agent Toolkit workflow configuration that serves as the target
+2. **Red Teaming Config**: A YAML file containing evaluator settings and attack scenarios
+3. **NAT CLI Red Team Command**: The CLI command that applies scenarios and executes evaluations
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)